src/components/originalSampleProcessing/blockProcessing/BlockProcessing.tsx

```diff
@@ -20,7 +20,7 @@ import Planner, { PlanChangedProps } from '../../planning/Planner';
 import createFormMachine from '../../../lib/machines/form/formMachine';
 import { stanCore } from '../../../lib/sdk';
 import WorkNumberSelect from '../../WorkNumberSelect';
-import columns from '../../dataTable/labwareColumns';
+import columns from '../../dataTableColumns/labwareColumns';
```

Comment:
multiple imports from the same file, thought prettier would have picked it up. Maybe change to `import { Columns, LabwareScanTableColumns } from '../../dataTableColumns/labwareColumns'`

Reply:
Think this happens in a few places

Reply:
Good finding!